### ISSUE TITLE
JAWS - navigation instruction for menu with aria-orientation="horizontal"

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/standards-support#204]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://codesandbox.io/s/stardust-ui-example-98ftk
> 2. Navigate by TAB into the menu item

### Expected result
Menu has attribute aria-orientation="horizontal", but user get message like for vertical menu.

### Actual result
JAWS narrate (take look on the instruction message):
"""
Editorials 
1 of 3
To move through items press up or down arrow.
"""

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
2019.1904.60

 
#### Operating System and version
Windows 10 Enterprise
Version 1903

#### Browser and version:
Chrome 74.0.3729.169

**Comment by JAWS-test** [comment]
JAWS ignores the aria-orientation attribute. It only works with sliders. This applies to various browsers and also older versions of JAWS.

**Comment by kolaps33** [comment]
If the JAWS ignores the aria-orientation attribute, why then providing the instruction message such "To move through items press up or down arrow." ? 
if such instruction is not valid for horizontal menu. How the JAWS user knows the menu orientation then?

**Comment by JAWS-test** [comment]
Yes, that's exactly the bug of JAWS. I just wanted to confirm that the problem is not only with Chrome and not only with role=menu, but of a general nature and should be solved by Freedom Scientific.

**Comment by jducrot** [comment]
@BenKeyFSI can you please confirm this is a JAWS bug?

**Comment by SuzanneTaylor** [comment]
JAWS 2020.2012 on a couple of different machines (and I'm told also JAWS 2021) is also ignoring aria-orientation for &lt;input type="range" /&gt; and reporting these as "left right slider"s. A test page that shows this is here: http://www.manateeroad.com/BugReports/InputRangeOrientation.html

**Comment by stevefaulkner** [comment]
tested with JAWS 2024.2310.70 https://codesandbox.io/s/stardust-ui-example-98ftk 
navigating to first item JAWS announces # of items, posinset and use left/right arrows to navigate

**Comment by stevefaulkner** [comment]
re-opening this as the support appears to be flaky:
works on https://codesandbox.io/p/sandbox/stardust-ui-example-98ftk
does not work on https://codepen.io/stevef/pen/XWGjZzw 

tested with: JAWS 2024.2312.53 and chrome Version 120.0.6099.200 (Official Build) (64-bit)
It is expected that JAWS would announce use of left/right arrows on both.
Orientation of the menu is exposed in acc tree on both test cases.

Also note that JAWS announces each menu item as "menu" not "menu item"

**Comment by stevefaulkner** [comment]
related https://github.com/FreedomScientific/standards-support/issues/411

**Comment by stevefaulkner** [comment]
A further wrinkle observed:
When the visual display of the controls is horizontal JAWS does announce the expected instruction (use left/right arrow keys), so it appears that aria-orientation is ignored in favour of visually based heuristics, implemented in JAWS

[test case no aria-orientation](https://codepen.io/stevef/pen/poYExVx)

**Comment by JeremiahRogersVispero** [comment]
Works as expected in JAWS 2026.2512.50
